### PR TITLE
Add flake8, lint target and lint project

### PR DESCRIPTION
**Pull request by ikornaselur**
Added flake8 and linted the project. Mainly it was unused imports or variables, but in some cases actual errors (`sys` wasn't imported in `kyber/__init__.py` and in `kyber/init.py` there was a catch block missing parenthesis around multiple exceptions to catch).

I don't believe any of the unused imports were required, though you might be able to correct me if I'm wrong. Tests still run, but I would like to be able to do a quick smoke test with a local dev kyber. How would I go ahead with installing this locally and globally to test? If not globally, what's the correct way to setup to try it out?

### CODE REVIEW

File: kyber/objects/secret.py [hunk 1]
```diff
@@ -41,10 +41,7 @@ def __unicode__(self):
         return unicode(repr(self))
 
     def has_key(self, k):
-        return self.obj['data'].has_key(k)
-
-    def pop(self, k, d=None):
-        return self.obj['data'].pop(k, d)
+        return k in self.obj['data']
```
Comment (steinnes) [inline kyber/objects/secret.py]: Why remove `pop`, and why not pass through the `has_key` method?  The intent of these methods is to mirror the standard interface of a `dict` (and `self.obj['data']` is a dict).

**Comment by steinnes** [comment]
Nice work :-)

The correct way to develop this locally is to run `make setup` then `source venv/bin/activate`, that way your `kb` entrypoint should be the local one you're editing.

File: kyber/objects/secret.py (hunk 1 above)
Comment (ikornaselur) [inline kyber/objects/secret.py]: Pop was defined twice, with the definition below, overwriting this one, being 

```
def pop(self, *args):
    return self.obj['data'].pop(*args)
```

`has_key` is deprecated and [has been removed in python3](https://docs.python.org/3.1/whatsnew/3.0.html#builtins) in favour of using `in`. Though I didn't realize the function here which used it is also `has_key()`. We can disable this rule though (the PR just has default rules at the moment). Thoughts?

**Comment by ikornaselur** [comment]
Okay, I tried running make setup and sourcing the venv and confirmed with `which kb` that it was using `kb` from the dev env, but I was getting errors when trying out kb at that point. I'll need to verify that it works on master and if so, verify that the linting done here isn't breaking anything.

**Pull request merged**